### PR TITLE
Add tree_time_[my_group] permalink support

### PR DESCRIPTION
**Pull request by fredj**
work in progress

### CODE REVIEW

File: core/src/script/CGXP/widgets/tree/LayerTree.js [hunk 1]
```diff
@@ -404,6 +411,17 @@ cgxp.tree.LayerTree = Ext.extend(Ext.tree.TreePanel, {
 
                 var timeWidget;
                 if (item.time) {
+                    if (this.initialState && this.initialState['time_' + item.name]) {
+                      var times = this.initialState['time_' + item.name].split('/');
+                      if (times.length === 2) {
```
Comment (sbrunner) [inline core/src/script/CGXP/widgets/tree/LayerTree.js]: As I remember we can also have only one time?

Comment (fredj) [inline core/src/script/CGXP/widgets/tree/LayerTree.js]: Yes, probably, but there's not example like this in the demo
I will check this, thanks.

Comment (fredj) [inline core/src/script/CGXP/widgets/tree/LayerTree.js]: > As I remember we can also have only one time?

Note that the min or the max can be set (not both min and max) with:
 - `tree_time_myGroupA=2006-01-01/`
 - `tree_time_myGroupA=/2016-01-01`

Comment (sbrunner) [inline core/src/script/CGXP/widgets/tree/LayerTree.js]: Yes but I think that's `tree_time_myGroupA=2016-01-01` should also be valid, to be verified :-)

Comment (sbrunner) [inline core/src/script/CGXP/widgets/tree/LayerTree.js]: The WMS time support many possibilities:
http://mapserver.org/ogc/wms_time.html#supported-time-requests
But with GeoMapFish we support only single and range :-)
https://github.com/camptocamp/c2cgeoportal/blob/master/doc/integrator/wmstime.rst#admin-interface

Comment (fredj) [inline core/src/script/CGXP/widgets/tree/LayerTree.js]: fixed

**Comment by sbrunner** [comment]
Looks good for me :-)

**Comment by fredj** [comment]
Thanks for the review !
I need to do more tests, I will finish it next week.

**Pull request merged**